libnetwork/slirp4netns/slirp4netns.go

```diff
@@ -685,6 +685,8 @@ func openSlirp4netnsPort(apiSocket, proto, hostip string, hostport, guestport ui
 	if _, err := conn.Write([]byte(fmt.Sprintf("%s\n", data))); err != nil {
 		return fmt.Errorf("cannot write to control socket %s: %w", apiSocket, err)
 	}
+	//nolint:errcheck // cast should never fail, if it does we can a nil deref panic
+	// and a stack trace on how we ended up here.
```

Comment:
This would typically not be a `nil` dereference (otherwise we would have panicked above):
> `panic: interface conversion: net.Conn is *net.TCPConn, not *net.UnixConn`

Reply:
ack thanks, silly me not actually checking the error that will be produced. I also added some more reason why we want this to the message...